ISISComputingGroup/EPICS-ioc
# Ticket1396 genie python get sample par not working

###Description of work

Re categorises some PVs so they're properly reported in genie_python using the get_sample_pars and get_beamline_pars. Also changes PARS:SCRIPT:NAME to PARS:SAMPLE:SCRIPT:NAME so that it is properly classified. The original has been left in as an alias

###To test

Ticket #1396 (rework)

###Acceptance criteria

First, update genie_python with the changes for PR https://github.com/ISISComputingGroup/genie_python/pull/40

1. Start EPICS
1. Verify that PARS:SCRIPT:NAME and PARS:SAMPLE:SCRIPT:NAME exist and have the same value
1. Run genie_python
    1. Call get_sample_pars() and get_beamline_pars(): the return dicts should be as expected
    1. Go to `C:\Instrument\Var\logs\genie_python` and verify that no errors are emitted as a result of the above calls

---

#### Code Review


### Functional Tests


### Final steps
